src/nox_poetry/sessions.py

```diff
@@ -224,8 +259,10 @@ def export_requirements(self) -> Path:
         digest = hashlib.blake2b(lockdata).hexdigest()
 
         if not hashfile.is_file() or hashfile.read_text() != digest:
-            constraints = to_constraints(self.poetry.export())
-            path.write_text(constraints)
+            requirements_text = self.poetry.export()
+            constraints = to_constraints(requirements_text)
+            pip_args = "\n".join(to_global_pip_args(requirements_text))
+            path.write_text("\n".join([pip_args, constraints]))
```

Comment:
I'm not sure if this data should be here or sent as arguments in the install command. Thoughts?